api/taxonomies/serializers.py

```diff
@@ -28,7 +29,20 @@ class TaxonomizableSerializerMixin(ser.Serializer):
         'subjects',
     ])
 
-    subjects = ser.SerializerMethodField()
+    def __init__(self, *args, **kwargs):
+        super(TaxonomizableSerializerMixin, self).__init__(*args, **kwargs)
+        request = kwargs['context']['request']
+
+        if self.expect_subjects_as_relationships(request):
+            self.fields['subjects'] = RelationshipField(
+                related_view=self.subjects_related_view,
+                related_view_kwargs=self.subjects_related_view_kwargs,
+                read_only=False,
+                many=True,
+                required=False,
+            )
+        else:
+            self.fields['subjects'] = ser.SerializerMethodField()
```

Comment:
WIP code to make subjects an attribute for early versions and a relationship for later versions.